cli/dts/lib.deno.unstable.d.ts

```diff
@@ -815,19 +815,28 @@ declare namespace Deno {
     windowChange: () => SignalStream;
   };
 
+  export type SetRawOptions = {
+    generateSignal: boolean;
+  };
+
   /** **UNSTABLE**: new API, yet to be vetted
    *
    * Set TTY to be under raw mode or not. In raw mode, characters are read and
    * returned as is, without being processed. All special processing of
    * characters by the terminal is disabled, including echoing input characters.
+   * Characters that correspond to a signal can optionally be enabled.
    * Reading from a TTY device in raw mode is faster than reading from a TTY
    * device in canonical mode.
    *
    * ```ts
-   * Deno.setRaw(myTTY.rid, true);
+   * Deno.setRaw(myTTY.rid, true, { generateSignal: true });
```

Comment:
`generateSignal` only does something when `mode == true`, right? That should probably be made clearer.